src/components/v5/common/ActionSidebar/partials/AmountField/AmountField.tsx

```diff
@@ -169,25 +171,36 @@ const AmountField: FC<AmountFieldProps> = ({
                           }}
                         >
                           <div className="flex items-center gap-1">
-                            <TokenIcon token={colonyToken} size="xs" />
-                            <span className="text-md">
-                              {colonyToken.symbol}
-                            </span>
+                            <TokenIcon
+                              token={colonyToken}
+                              size="xs"
+                              className="mr-1.5"
+                            />
+                            <HoverWidthWrapper
+                              hoverClassName="block"
+                              hoverElement="span"
+                            >
+                              <span className="text-md w-0 overflow-visible">
+                                {colonyToken.symbol}
+                              </span>
+                            </HoverWidthWrapper>
                           </div>
                           {tokenBalance && (
-                            <span className="text-sm text-gray-400">
-                              {formatMessage({
-                                id: 'actionSidebar.availableFunds',
-                              })}
-                              {': '}
-                              <Numeral
-                                value={tokenBalance}
-                                decimals={getTokenDecimalsWithFallback(
-                                  colonyToken?.decimals,
-                                )}
-                              />{' '}
-                              {colonyToken.symbol}
-                            </span>
+                            <HoverWidthWrapper hoverElement="span">
```

Comment:
Can you please check if wrapping the whole `button` with this component gives the same effect?